lib/rex/proto/rfb/constants.rb

```diff
@@ -13,39 +13,69 @@
 ##
 
 module Rex
-module Proto
-module RFB::Constants
+  module Proto
+    module RFB::Constants
+      DefaultPort = 5900
 
-DefaultPort = 5900
+      # Version information
+      MajorVersions = [3, 4, 5]
```

Comment:
I'll accept this but please update your description of this PR since you said your seeing it in something but didn't specify where you are seeing it, just left it as an incomplete sentence 😄